# RemoteControlStatus.msg - 遥控器状态反馈
# 遥控器状态
uint8 swa               # 拨杆A状态
uint8 swb               # 拨杆B状态
uint8 swc               # 拨杆C状态
uint8 swd               # 拨杆D状态
int8 right_lr           # 右手左右摇杆
int8 right_ud           # 右手上下摇杆
int8 left_lr            # 左手左右摇杆
int8 left_ud            # 左手上下摇杆
int8 left_vra           # 左手旋钮